docker-compose.yml

```diff
@@ -80,18 +82,14 @@ services:
       MERCURE_SUBSCRIBER_JWT_KEY: ${MERCURE_SUBSCRIBER_JWT_KEY:-!ChangeMe!}
       MERCURE_EXTRA_DIRECTIVES: demo
     ports:
-      # HTTP
+      # frontend
```

Comment:
Isn't this comment wrong? All services are served via port 3000 now, not just the frontend.